lms/models/lti_role.py

```diff
@@ -18,6 +18,7 @@ class RoleType(str, Enum):
     INSTRUCTOR = "instructor"
     LEARNER = "learner"
     ADMIN = "admin"
+    NONE = "none"
```

Comment:
Added a new "type".

This works because LTI roles are first parsed and assigned a pair of type and scope and then, based on those we pick our behaviours, which are basically only:

- learner
- instructor
- admin


RoleType and RoleScope are explicitly compared here:

https://github.com/hypothesis/lms/blob/null-role/lms/models/lti_user.py#L87

so to ignore the `None` ones we don't have to change anything else.